app/services/rbac.rb

```diff
@@ -77,8 +77,7 @@ def valid_inventory_groups_definition?(definition)
       JSON.parse(definition.value).instance_of?(Array) &&
         definition.operation == 'in' &&
         definition.key == 'group.id'
-    rescue JSON::ParserError => e
-      Rails.logger.error("Invalid JSON value received: #{e}")
```

Comment:
I would keep this logging, but rather move it into upper level.

Reply:
Keep the logging, but move it to info maybe